app/controllers/scholarships_controller.rb

```diff
@@ -42,14 +42,21 @@ def applications
   end
 
   def auto_complete_for_scholarship_title
-		if params[:scholarship][:title].is_integer?
-			@scholarships = [Scholarship.find(params[:scholarship][:title])]
+    term = params[:term] || params[:scholarship][:title]
+		if term.is_integer?
+			@scholarships = [Scholarship.find(term)]
 		else
-	    @scholarships = Scholarship.find(:all, :conditions => ["LOWER(title) LIKE ?", "%#{params[:scholarship][:title].to_s.downcase}%"], :limit => 20)
+	    @scholarships = Scholarship.find(:all, :conditions => ["LOWER(title) LIKE ?", "%#{term.to_s.downcase}%"], :limit => 20)
 		end
-    render :partial => "shared/auto_complete_scholarship_title", 
-            :object => @scholarships, 
-            :locals => { :highlight_phrase => params[:scholarship][:title] }
+    render :json => @scholarships.map { |result| 
```

Comment:
Avoid using {...} for multi-line blocks.<br>Use the new Ruby 1.9 hash syntax.<br>Trailing whitespace detected.